_data/vendors.yml

```diff
@@ -158,6 +158,16 @@
   pricing_source: https://github.com/pricing
   updated_at: 2020-04-14
 
+- name: Grammarly
+  url: https://www.grammarly.com
+  base_pricing: $12.50 per u/m
+  sso_pricing: $583.33 per u/m [^grammarly]
+  percent_increase: 5000%
```

Comment:
Should be 4567%

Reply:
maths is hard lol

Reply:
We really should add the Contributors.md from the parent repository and maybe a link to [something like this](https://www.omnicalculator.com/math/percentage-increase) to make it easier for submitters.

Reply:
true. 
i do not code though. so i can't built anything like that.

Reply:
but let's move this to somewhere else #10, maybe?